apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html

```diff
@@ -549,35 +552,35 @@ <h4 class="font-16 strong d-flex align-items-center mb-3 plan-summary-heading">
                                             <div class="form-group">
                                                 <input-field
                                                     [type]="'text'"
-                                                    [name]="'billingName'"
+                                                    [name]="'subscriptionBillingName' + currentTimestamp"
                                                     [label]="localeData?.billing_name + '*'"
                                                     [placeholder]="localeData?.billing_name"
                                                     formControlName="billingName"
                                                     [showError]="
                                                         isFormSubmitted && !secondStepForm.get('billingName')?.value
                                                     "
-                                                    [autocomplete]="'off'"
+                                                    [autocomplete]="'subscriptionBillingName' + currentTimestamp"
```

Comment:
what's the use of this????

Reply:
Autocomplete uses a string value, so I assigned a dynamic value.